### ISSUE TITLE
evaluating on race dataset with checkpoints fine tuned on roberta with fairseq

### ISSUE THREAD

**Issue by qshi95** [huggingface/transformers#1576]
## ❓ Questions & Help

I fine tuned a model on race dataset with reberta, following the fairseq instruction, got the result:
| epoch 004 | valid on 'valid' subset:  | loss 0.913 | nll_loss 0.003 | ppl 1.00 | num_updates 21849 | best_accuracy 0.846563 | accuracy 0.836129
| epoch 004 | valid on 'valid' subset:  | loss 0.913 | nll_loss 0.003 | ppl 1.00 | num_updates 21849 | best_accuracy 0.846563 | accuracy 0.836129
| epoch 004 | valid on 'valid' subset:  | loss 0.913 | nll_loss 0.003 | ppl 1.00 | num_updates 21849 | best_accuracy 0.846563 | accuracy 0.836129
| epoch 004 | valid on 'valid' subset:  | loss 0.913 | nll_loss 0.003 | ppl 1.00 | num_updates 21849 | best_accuracy 0.846563 | accuracy 0.836129
| saved checkpoint checkpoints/checkpoint4.pt (epoch 4 @ 21849 updates) (writing took 145.8246190547943 seconds)
| done training in 76377.9 seconds

But I load the weight to the transformers with convert_roberta_original_pytorch_checkpoint_to_pytorch script:

python convert_roberta_original_pytorch_checkpoint_to_pytorch.py --roberta_checkpoint_path ../pytorch-transformers-master/data/roberta-best-checkpoint/ --pytorch_dump_folder_path ../pytorch-transformers-master/data/roberta-best-checkpoint/

then evaluate on RACE dataset, I got terrible results on dev set:
model           =data/models_roberta_race/
total batch size=8
train num epochs=5
fp16            =False
max seq length  =512
eval_acc = 0.4808676079394311
eval_loss = 1.352066347319484

and on test set:
model           =data/models_roberta_race/
total batch size=8
train num epochs=5
fp16            =False
max seq length  =512
eval_acc = 0.6015403323875153
eval_loss = 1.3087183478393092

I don't know why. Could anyone can help? Thank you!

**Comment by PantherYan** [comment]
Do you get any improvement? 
The ACC of eval and test has a huge gap.

**Comment by pratyay-banerjee** [comment]
same here, --classification-head when converting the models

**Comment by sgdgp** [comment]
I am unable to reproduce the results on the RACE dataset. If anyone has been able to reproduce it, could you kindly share the weights of the fine-tuned model ?
